bugsnag-plugin-android-anr/src/test/java/com/bugsnag/android/AnrDetailsCollectorTest.kt

```diff
@@ -70,9 +58,4 @@ class AnrDetailsCollectorTest {
         assertEquals(stateInfo, captureProcessErrorState)
     }
 
-    @Test
-    fun anrDetailsAltered() {
-        collector.addErrorStateInfo(event, stateInfo)
-        assertEquals(stateInfo.shortMsg.replace("ANR", ""), event.exceptionMessage)
-    }
```

Comment:
This still appears to be a valid test?

Reply:
Updated test case to verify against the new event interface.